<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="Title1">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>평면 공간에 존재하는 2개의 도선에 의한 자기장의 세기를 계산합니다.</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>50</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="text">
     <string>도선에 흐르는 전류의 세기와 방향, 도선이 존재하는 x축의 값을 지정해주세요.</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title2_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>70</y>
      <width>311</width>
      <height>16</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>6</number>
    </property>
    <property name="text">
     <string>그리고 자기장의 세기를 측정할 곳의 위치를 정해주세요.</string>
    </property>
   </widget>
   <widget class="Line" name="axisX">
    <property name="geometry">
     <rect>
      <x>-170</x>
      <y>390</y>
      <width>1251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>2</number>
    </property>
    <property name="midLineWidth">
     <number>2</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line1">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>81</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line2">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>120</y>
      <width>81</width>
      <height>581</height>
     </rect>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>5</number>
    </property>
    <property name="midLineWidth">
     <number>5</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>460</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>B = k(상수) * I</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="line1_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>140</y>
      <width>261</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>도선 1</string>
    </property>
    <widget class="QGroupBox" name="Direct1">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>방향</string>
     </property>
     <widget class="QRadioButton" name="up1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>위(+ y)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="down1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>아래(- y)</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>세기 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>d</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inten1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>위치 :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pos1">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>140</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>도선 2</string>
    </property>
    <widget class="QGroupBox" name="Direct2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>221</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>방향</string>
     </property>
     <widget class="QRadioButton" name="up2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>위(+ y)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="down2">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>30</y>
        <width>90</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>아래(- y)</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>세기 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>31</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>위치 :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pos2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>30</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>d</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inten2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>21</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>I</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>230</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>자기장을 측정할 위치</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="calpos">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>260</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ResultButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>300</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>계산</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>271</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>결과</string>
    </property>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>방향 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>세기 :</string>
     </property>
    </widget>
    <widget class="QLabel" name="DirectResult">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
    <widget class="QLabel" name="IntenResult">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>161</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>?</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>260</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>d</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>90</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>도선 1을 도선 2보다 위치 값을 작게 해주세요.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_15">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>600</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ErrorMessage : </string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorMsg">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>600</y>
      <width>381</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>None</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>560</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>세기는 세 번째 소수점에서 반올림 됩니다.</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
